db/sql/0_msar.sql

```diff
@@ -2192,6 +2233,18 @@ BEGIN
       msar.process_col_alter_jsonb(tab_id, col_alters)
     );
   END IF;
+
+  -- Here, we perform all description-changing alterations.
+  PERFORM __msar.comment_on_column(
+    tab_id := tab_id,
+    col_id := (col_alter->>'attnum')::integer,
```

Comment:
I like the `AS x(attnum integer, description text)` syntax, but I couldn't figure out how to use it and to be able to do `jsonb_key_exists(col_alter, 'description');` lower down as well. Am I missing something?